### ISSUE TITLE
Effect of hyperparameter L

### ISSUE THREAD

**Issue by bangluqman** [hkchengrex/CascadePSP#48]
![Screenshot from 2021-06-24 16-35-16](https://user-images.githubusercontent.com/19197777/123222002-7a1c5b00-d50a-11eb-9e0c-00cb0b1d8437.png)

Hello, I try to testing with the same L = 900. 
I got the result like an original image. Then, I try to decrease L, I got the result but the looks like mixing with the original image.
Why its happen?

**Comment by hkchengrex** [comment]
That should not happen. Can you share your inputs?

**Comment by bangluqman** [comment]
![Screenshot from 2021-06-24 17-02-52](https://user-images.githubusercontent.com/19197777/123225852-16942c80-d50e-11eb-9d1e-3b1f42e96730.png)

This is my inputs. I use the output from DeepLabV3+ as input. It's the same as input in figure 14 in the paper.

**Comment by hkchengrex** [comment]
Are you using our code demo? The inputs look fine and I cannot think of any reason for the output to look like that without additional info.

**Comment by bangluqman** [comment]
No, I try training by myself. and use my trained model.
Due to the limitation of my GPU memory, I use batch= 4  and not use nn.DataParallel.

If I use code demo  refine.Refiner, it can work very well.

**Comment by hkchengrex** [comment]
We haven't tried smaller batch sizes but maybe it is crucial to have larger batches. You can try to reduce memory consumption with PyTorch's AMP or gradient accumulation.
